video/decode/vd_lavc.c

```diff
@@ -247,6 +248,9 @@ const struct autoprobe_info hwdec_autoprobe_info[] = {
     {"mediacodec-copy", HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
     {"videotoolbox",    HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
     {"videotoolbox-copy", HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
+    {"cuda",            HWDEC_FLAG_DENYLIST },
+    {"cuda-copy",       HWDEC_FLAG_DENYLIST },
+    {"v4l2m2m-copy",    HWDEC_FLAG_DENYLIST },
```

Comment:
should we really do this when the docs are pretty clear that people should use `auto-safe`?

also v4l2m2m-copy doesn't "prevent playback fromhappening at all" on my system

Reply:
On my machine (normal desktop), trying to use v4l2m2m-copy resulted in the stream abruptly hitting EOF.

But yes, I discussed this in the commit description (I guess I should have copied it to the PR description). We'd need to change Ctrl+h to use `auto-safe` by default.